config/database.yml

```diff
@@ -20,6 +20,8 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  user: corcovado
```

Comment:
@drippdropp Hey Daniel, I wouldn't think that we should be committing additions like this to the general codebase? 🤔  How do we avoid this?